Sources/MapboxMaps/Ornaments/OrnamentsManager.swift

```diff
@@ -103,10 +103,10 @@ public class OrnamentsManager: NSObject {
         // Activate new constraints
         NSLayoutConstraint.activate(constraints)
 
-        logoView.isHidden = !options.logo._isVisible
+        logoView.isHidden = options.logo.visibility == .hidden
```

Comment:
Is this a protocol? Wondering if we can use `_visibility`, maintaining the `_`.

Reply:
Right now it's a struct, but could go back to using a protocol in order to keep the underscore.